London-10-Abubakar-Meigag-HTML/CSS-RECIPE COURSEWORK-week2

Abubakar-Meigag opened the pull request:
<!--

You must title your PR like this:

COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME

For example,

NW4 | Carol Owen | HTML-CSS-Coursework-Week1

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

I'm still no done but if there any comment please let me know

Abubakar-Meigag committed:
create html and css

. create html and css
. write all semantic html
. add some main css style

Abubakar-Meigag committed:
start to made layout

I choose Grid to made layout for my web
add so main grid layout
change font color

Abubakar-Meigag committed:
layout

. add and grid-areas for layout
. drawing all layout with grid template-areas.

Abubakar-Meigag committed:
add content

add content and style css for the web

ibrahimparvez2 commented:
Please fix title

ibrahimparvez2 commented:
please see readme file instructions:
EXPECTATION: Create a new recipe page for your favourite meal and how to make it and where it comes from making sure you put into practice the techniques learnt throughout the week.

Abubakar-Meigag committed:
edit

. Edit all wrong content
. Edit title
. add file for img
.add content html and style it
